addons/website/static/tests/tours/link_tools.js

```diff
@@ -14,6 +14,18 @@ const clickOnImgStep = {
     run: "click",
 };
 
+const clickOnLink = (selector) => ({
+    content: "Click on the link",
+    trigger: selector,
+    async run(helpers) {
+        await helpers.click();
+        const el = this.anchor;
+        const sel = el.ownerDocument.getSelection();
+        sel.collapse(el, 0);
+        el.focus();
+    },
+});
+
```

Comment:
We have `openLinkPopup` a util method you can use that in place of creating new.